dashboard/app/views/levels/editors/fields/_authored_hints.haml

```diff
@@ -67,6 +67,15 @@
             'markdown',
             {
               callback: function(editor, change) {
+                //Looking for `.markdown_preview div --> div --> [XML Blockly divs]` structure
+                //gets .markdown_preview div
```

Comment:
nit: comments should probabl y lo

Reply:
@breville I might be missing something, but this comment this looks incomplete to me? Something got cut off?

Reply:
Oops, not sure what happened.  Anyway, was suggesting a space after each `//`, a `.` at the end of each sentence, and a blank line after the relevant code before the next comment.

Reply:
Oh, it's the same comment below, so I'll just remove this thread.